tools/cmdline/src/cf-client-utils.js

```diff
@@ -26,8 +26,11 @@ const deleteApplication = (name, deleteRoute) => {
   return execute(`cf delete -f ${deleteRoute ? '-r' : ''} ${name}`);
 };
 
-const createServiceInstance = (service, plan, serviceName) => {
-  return execute(`cf create-service ${service} ${plan} ${serviceName}`);
```

Comment:
We can have this as: 
`cf create-service ${service} ${plan} ${serviceName} ${ parameters ? ' -c ${parameters}' : '' }`